### DISCUSSION TITLE
The trace information is not displayed completely where service call out by http

### DISCUSSION THREAD

**Discussion by QQQJoker** [apache/skywalking#11452] [Q&A]
I have encountered an issue while using Skywalking as a monitoring platform. The problem is that when a service makes an external HTTP call, the trace information is not displayed completely. For example, if A calls B via an HTTP service, and B accesses data using MyBatis. However, when checking the trace, only one entry and exit for service A are shown, without an entry for service B. Has anyone encountered a similar issue and found a solution?

**Comment by wu-sheng** [comment]
Most likely, the HTTP server framework of your service B is not supported.

**Reply by QQQJoker** [reply]
When I directly call service B using Postman, the B side can trace the information of this API call. However, when service A calls service B, the trace information of this API call is not displayed on the B side. When calling from service A to service B externally, the trace information is not displayed, but when directly calling service B using Postman, it is displayed.

**Reply by wu-sheng** [reply]
This doesn't make sense. Server side tracing has no relationship with the caller.

**Reply by QQQJoker** [reply]
![image](https://github.com/apache/skywalking/assets/16871706/e49c4e70-e2ae-4f8f-b480-2c283f629af8)

**Reply by wu-sheng** [reply]
I don't mean you are not telling the truth. I mean, I can't explain why.

**Reply by wu-sheng** [reply]
You have to debug on your own. There isn't known issue about this.

**Reply by QQQJoker** [reply]
ok, thanks
